ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md

```diff
@@ -81,3 +82,40 @@ Where:
 * you can check the mapping between partition IDs and partition names using `gdisk -l` command
 
 If you insert the SD card into the DragonBoard410c and set the switch S6-2 (on the back) to ON, you should now be able to boot from SD card.
+
+# 4) Booting Debian image
+
+Download boot and root images from:
+  - **Release:** http://builds.96boards.org/releases/dragonboard410c/linaro/debian/latest/
+  - **Snapshot:** http://builds.96boards.org/releases/dragonboard410c/linaro/debian/latest/
+
+  For Example:
+  ```shell
+  $ wget http://builds.96boards.org/releases/dragonboard410c/linaro/debian/latest/boot-sdcard-linaro-stretch-qcom-snapdragon-arm64-20171016-283.img.gz
+  $ wget http://builds.96boards.org/releases/dragonboard410c/linaro/debian/latest/linaro-stretch-alip-qcom-snapdragon-arm64-20171016-283.img.gz
+  ```
+  And then extract the images
+  ```shell
+  $ gunzip boot-sdcard-linaro-stretch-qcom-snapdragon-arm64-20171016-283.img.gz
+  $ gunzip linaro-stretch-alip-qcom-snapdragon-arm64-20171016-283.img.gz
+  ```
+
+Since we are already downloading the boot-sdcard-xxx.img.gz, we do not need to modify the boot parameters.
+
+You can now write the boot and rootfs image into the SD card:
+
+```shell
+$ sudo dd if=boot-sdcard-XXX.img of=/dev/XXX7
+$ sudo simg2img linaro-stretch-alip-XXX.img /dev/XXX9
+```
+NOTE: you may need to install the ```simg2img``` package using:
+```shell
+$ sudo apt install simg2img
```

Comment:
nope. it's in android-tools-fsutils

Reply:
seems to be simg2img on debian, ubuntu(zesty artful bionic)
i'll add that too